### PR TITLE
Create Address Validation 3.0.md

### PR DESCRIPTION
**Pull request by sli2**
Migrating documentation for 3.0 Address Validation from website so we can update website to 3.1 documentation.

### CODE REVIEW

File: Address Validation/3.0.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+All parameters are case sensitive. api_key, a street parameter, and either zip or state is required for a Address Validation request.
+
+| KEY     | Description | Examples |
+| ------- | ---- | ---- |
+| api_key | See https://pro.whitepages.com/developer to acquire an API key. REQUIRED | |
+| street_line_1 | Number and street name REQUIRED | 2808 Nero Blvd |
+| street_line_2 | Apartment or other additional address information | Apt 265, Box 34Rs |
+| city | City name. REQUIRED if there is no input postal_code parameter | Seattle, New York |
+| postal_code | 5 or 9 digit US or 6 digit Canadian zip code. REQUIRED if there is no input city and state_code parameters | 92019 or S3D 3F3 |
```
Comment (sakulich) [inline Address Validation/3.0.md]: it is global endpoint. just local country postal code we need generic message. it is not just US/CA

**Comment by sli2** [comment]
@raszi thanks for adding more edits, can you please publish this? or do I need to do anything else?

File: Address Validation/3.0.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+| `zip4` | string | 4-digit US zipcode extension. |
+| `state_code` | string | Normalized 2 character state/province code in US/CA or state name internationally. Examples: “WA” for Washington or “BC” for British Columbia. |
+| `country_code` | string | Country code (ISO 3166-1 alpha-2), Example: “US”. |
+| `lat_long` | object | Latitude and longitude associated with the returned address. |
+| `warnings` | array | An array containing one or more warning message strings. |
+| `error` | object | An error object which contains name string and message string. |
+
+### diagnostics array
+
+| Field | Type | Description |
```
Comment (raszi) [inline Address Validation/3.0.md]: JSON array types has no fields. The language should be fixed here. Type is always string, so I can imagine something like:

| Value | Description |
| ----- |  ---- |
| `"Validated"` | Full address validated. |

**Comment by sli2** [comment]
Made the changes... @raszi

**Pull request merged**